### ISSUE TITLE
Separators need conversion as well

### ISSUE THREAD

**Issue by saeedalipoor** [persian-tools/persian-tools#110]
https://github.com/persian-tools/persian-tools/blob/a023ef26b9d4cd975a08714b0b5c5a68227a775c/src/modules/digits/index.ts#L20
as `Number.toLocaleString` function, the number separators should convert too.
`,` should convert to `٬` and `.` should convert to `٫`.
BTW Why didn't we use the `toLocaleString` function instead of `replace`?

**Comment by ali-master** [comment]
Hi Saeed Jan,

@psparsa created a PR and He refactored all digits functions and he can implement this feature. Or if you want, help us by creating a PR because the Persian-Tools is yours.

**Comment by parsa-vxt** [comment]
> BTW Why didn't we use the `toLocaleString` function instead of `replace`?

the `toLocaleString` is one of the Number object methods, we can only use that for converting English numbers to Farsi, Arabic & etc; and the second problem, that's possible our input be a string that contains the English digits, and we wanna change those digits to Farsi.. at this case the `Number.toLocaleString` is not practical.

**_e.g:_**

**input:
10,000 ﷼**

**output:
۱۰,۰۰۰ ﷼**

**Comment by saeedalipoor** [comment]
Hi @ali-master,
I would be happy to open a PR as soon as possible.

Thanks, @psparsa
What do you think of changing the replace function to something like this:
```
str.replace(/\d+/g, match=>Number(match).toLocaleString('fa-ir'))
```

**Comment by psparsa** [comment]
> What do you think of changing the replace function to something like this:
>
> str.replace(/\d+/g, match=>Number(match).toLocaleString('fa-ir'))

this won't work correctly, take a look at these examples:

```
Number("09123456789"); //9123456789
Number("۰۱۲۳"); //NaN
Number("X: 12, Y: 84"); //NaN
```

**Comment by parsa-vxt** [comment]
> What do you think of changing the replace function to something like this:
> 
> str.replace(/\d+/g, match=>Number(match).toLocaleString('fa-ir'))

seems correct and will works; but just for converting english digits to ....
in my last PR i refactored the digits converters... now it's like this:

**_EnToFa:_**
```String(value).replace(enDigitsRegex, (char) => `${faNums[Number(char)]}`);```

**Comment by ali-master** [comment]
I'm in

**Comment by saeedalipoor** [comment]
Yes, I proposed that for converting English digits to Farsi or other locales digits.
It replaces numbers instead of replacing digits one by one.

**Comment by ali-master** [comment]
@saeedalipoor Jan, any news or updates on this issue?
